mail_tracking/models/res_company.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    show_aliases_mail_tracking = fields.Boolean(
```

Comment:
Please use a better nomenclature with precedence to the "section" (module) and avoid plurals:

```suggestion
    mail_tracking_show_alias = fields.Boolean(
```

Reply:
I see that is still keeping the plural. Better to not include it for avoiding typos on development.

Reply:
done

Reply:
Thanks. It seems there's an error in CI. And what about the use case expanded info for judging if we should do it the standard?